src/main/scala/tilelink/Arbiter.scala

```diff
@@ -18,12 +18,12 @@ object TLArbiter
   val roundRobin: Policy = (width, valids, select) => if (width == 1) UInt(1, width=1) else {
     val valid = valids(width-1, 0)
     assert (valid === valids)
-    val mask = RegInit(~UInt(0, width=width))
-    val filter = Cat(valid & ~mask, valid)
-    val unready = (rightOR(filter, width*2, width) >> 1) | (mask << width)
+    val maskn = RegInit(UInt(0, width=width)) // val mask = RegInit(~UInt(0, width=width)) fails with AsyncReset
```

Comment:
@jackkoenig @azidar this "fails with AsyncReset comment" is presumably something where it can't tell that ~UInt(const) is also a const. I assume that is still an issue and I should not try to undo this change?

Reply:
In the mean time, why don't we do `val mask = RegInit(((BigInt(1) << width)-1).U(width.W))`, to avoid the other RTL changes while keeping the constant constant?